[docs] Format fixes Integrations pages (#14015)
yugabyte/yugabyte-db: docs/content/preview/integrations/django-rest-framework.md

> Before you can start using Django, ensure that you have the following installed and configured:

Reviewer: ```suggestion
To use Django with YugabyteDB, you need the following:
```

Reviewer: same in flyway, gorm too.
